### PR TITLE
Missing Permissions check

### PR DESCRIPTION
**Pull request by OGNova**
Does exactly what the name says. This will check if the bot is missing any permissions that it requires for a command to be run, drawn from `cmd.conf.botPerms`.

### CODE REVIEW

**Review by MrJacz — CHANGES_REQUESTED** [review]
I'd prefer if you made the checking into a function (Class method)

**Comment by MrJacz** [comment]
**Disclaimer: I'm not saying use Klasa!**
Might wanna look at [**Klasa missing perms inhibitor**](https://github.com/dirigeants/klasa/blob/master/src/inhibitors/missingBotPermissions.js) `cmd.botPerms` is an instance of Permissions class (this doesn't matter but just saying) should use some methods there to spruce the display of the message

**Review by MrJacz — APPROVED** [review]
I'll do my requested changes in a separate PR later...
@YorkAARGH This can be merged.

**Review by kyranet — APPROVED** [review]

File: events/message.js [hunk 1]
```diff
@@ -70,6 +70,12 @@ module.exports = class {
     
     this.client.logger.log(`${this.client.config.permLevels.find(l => l.level === level).name} ${message.author.username} (${message.author.id}) ran command ${cmd.help.name}`, "cmd");
     
+    if (message.channel.type === "text") {
```
Comment (YorkAARGH) [inline events/message.js]: This channel type check isn't required at all as Misaki actively ignores any channel that doesn't have a guild object.

Comment (OGNova) [reply events/message.js]: She doesn't respond in DMs?

Comment (YorkAARGH) [reply events/message.js]: [What do you think?](https://github.com/NotAWeebDev/Misaki/blob/master/events/message.js#L9)

**Review by DevYukine — APPROVED** [review]

**Pull request merged**